internal/engine/eval/vulncheck/pkgdb.go

```diff
@@ -44,6 +47,10 @@ func urlFromEndpointAndPaths(
 	return u, nil
 }
 
+type formatterMeta struct {
+	missingInRegistry bool
```

Comment:
I think we should rather have an error here that we could `errors.Is` on instead of adding bespoke methods that we need to implement over and over.

Or, we could implement them for the formatterMeta only.